omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/MainResourceController.java

```diff
@@ -123,9 +123,18 @@ public Object update(@PathVariable("resource") String resource, @PathVariable("u
 	        throws ResponseException {
 		baseUriSetup.setup(request);
 		RequestContext context = RestUtil.getRequestContext(request, response);
-		Updatable res = (Updatable) restService.getResourceByName(buildResourceName(resource));
-		Object updated = res.update(uuid, post, context);
-		return RestUtil.updated(response, updated);
+		
+		if (post.get("deleted") != null && "false".equals(post.get("deleted")) && post.size() == 1) {
+			//This branching provides for unvoiding or unretiring a resource 
```

Comment:
This comment is obvious from context, so you don't need to explicitly state this.

Reply:
I deleted the comment